src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -230,6 +230,11 @@ public static bool IsOrClosedOverATypeFromAssemblies(TypeSymbol symbol, Immutabl
 
                     do
                     {
+                        if (namedType.IsTupleType)
```

Comment:
This is NoPia specific method and its name uses terms relevant to the feature.

---
In reply to: [74150130](https://github.com/dotnet/roslyn/pull/13024#discussion_r74150130) [](ancestors = 74150130)